microprofile/WFLY-14940_upgrade-microprofile-health-3.1.adoc

```diff
@@ -0,0 +1,69 @@
+= Upgrade MicroProfile Health to 3.1
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          microprofile,health,observability
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+
+MicroProfile 4.1 provides a minor upgrade for https://github.com/eclipse/microprofile-health/releases/tag/3.1[MicroProfile Health 3.1] with changes described in its https://download.eclipse.org/microprofile/microprofile-health-3.1/microprofile-health-spec-3.1.html#release_notes_3_1[Release Notes].
+
+This component upgrade also covers the upgrade of smallrye-health 3.1.1 that implements MicroProfile Health 3.1.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-14940[WFLY-14040]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1763[EAP7-1763]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:fburzigo@redhat.com[Fabio Burzigotti]
+
+=== Testing By
+
+[X] Engineering
+[ ] QE
```

Comment:
Just a minor issue with formatting, here. In the rendered adoc file I can see both the checks being on the same line and maybe the `X` should be in lower case.

Reply:
Not to be pedantic, and I don't care actually since it is readable, but they're still on the same line. Maybe a blank row between the two checks could help.